Add info about failed PayJoin txs (#911)
zkSNACKs/WasabiDoc: docs/using-wasabi/PayJoin.md

1. Load a wallet and open the `Send` tab.

2. Request from the receiver a [BIP21 Bitcoin URI](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) with the flag `pj=` and paste it into the address field of the `Send` tab.

> (Eg. `bitcoin:bc1q52089pawz7vqmyafzpf9xuklpmzh0qhszqx23h?amount=0.07361908&pj=https://btcpay.kukks.org/BTC/pj`

Reviewer: How about a testnet uri
```suggestion
(E.g. `bitcoin:tb1q0382a3m2jzvyk5lkea5h5jcht88xa6l0jufgwx?amount=0.00010727&pj=https://demo.payjoin.kukks.org/BTC/pj`)
```

3. Select the coins you want to send.

4. Verify the information is correct, type in the password, and press `Send`.

![](/PayJoinSend.png)

:::warning The success of a PayJoin transaction is not something that depends on Wasabi.

> It may happen that the transaction is broadcasted with success, but that it is not a PayJoin.

Reviewer: ```suggestion
It may happen that the transaction is broadcast with success, but that it is not a PayJoin.
```

This can happen for several reasons including:

> - The receiver did not have any utxos to contribute towards a PayJoin

Reviewer 2: ```suggestion
- The receiver did not have any utxos to contribute towards a PayJoin.
```

> - Your wallet does not use the same format as the receiver's (essential to not raise suspicion to [surveillance companies](/why-wasabi/TransactionSurveillanceCompanies.md))

Reviewer 2: ```suggestion
- Your wallet does not use the same address format as the receiver's.
```

> - The PayJoin server is not available

Reviewer 2: ```suggestion
- The PayJoin server is not available.
```
